gno.land/pkg/gnoweb/handler.go

```diff
@@ -183,6 +187,103 @@ func (h *WebHandler) GetRealmView(gnourl *weburl.GnoURL) (int, *components.View)
 	})
 }
 
+// GetUserView returns the user profile view for a given GnoURL.
+func (h *WebHandler) GetUserView(gnourl *weburl.GnoURL) (int, *components.View) {
+	username := strings.TrimPrefix(gnourl.Path, "/u/")
```

Comment:
It works now, but I believe we will have subpages for users, pagination, etc. I'm not sure if everything will rely on queries, but it's fine to use the current routing mechanism for now.